### DISCUSSION TITLE
Parsing Left-Associative Operators of Equal Precedence

### DISCUSSION THREAD

**Discussion by ghost** [zesterer/chumsky#73] [Q&A]
It's all in the name. With a single operator, it's easy enough to just write it as a repetition and then fold over the resultant `Vec`, but that doesn't work when there are multiple operators of the same precedence. What is the best way to do this? As an example, let's say I want to parse the `+` and `-` operators, which have equal precedence and are left-associative.

**Comment by zesterer** [comment]
Hey, apologies about not noticing this until now.

I believe that [binary operator](https://github.com/zesterer/chumsky/blob/master/tutorial.md#parsing-binary-operators) section in the tutorial *should* cover this. If not, I'm happy to help!
